dependencies.gradle

```diff
@@ -0,0 +1,44 @@
+def versions = [
+    kotlinVersion                : '1.2.21',
```

Comment:
might want to bump the kotlin version?

Reply:
I'll leave this to a follow-up, partially because buck currently has some issues with `1.2.25`